frontend/src/theme/palette/index.ts

```diff
@@ -1,9 +1,22 @@
 import { Color } from "@mui/material";
 import { ThemeOptions } from "@mui/material/styles";
 
-export const palette = (mode: "light" | "dark"): ThemeOptions["palette"] => ({
+const getPalette = (mode: "light" | "dark" | "janus") => {
+  switch (mode) {
+    case "light":
+      return light;
+    case "dark":
+      return dark;
+    case "janus":
+      return janus;
+    default:
+      return light;
+  }
+};
+
+export const palette = (mode: "light" | "dark" | "janus"): ThemeOptions["palette"] => ({
   mode,
-  ...(mode === "light" ? light : dark),
```

Comment:
can revert back to this old solution now that we directly pass in janus theme in ThemeProvider component